cmd/iam-store.go

```diff
@@ -1277,9 +1284,10 @@ func (store *IAMStoreSys) GetUserInfo(name string) (u madmin.UserInfo, err error
 		// If the user has a mapped policy or is a member of a group, we
 		// return that info. Otherwise we return error.
 		var groups []string
+		var mappedSysUser, mappedSysGroup string
```

Comment:
There is also "GetUsers()" function in this file that me may need to update.

Reply:
It was changed: https://github.com/panasasinc/minio/pull/27/files#diff-b36131f5f2912eeb5ef4c601441d61a1f48cc621a01c5ed7cedc9fad2ee3b6b6R1254